tests/functional/test_inference_per_framework.py

```diff
@@ -78,7 +78,7 @@
     R_FAIL_CLASSIFICATION_VALIDATION_HOOKS,
     R_PREDICT_SPARSE,
     R_TRANSFORM_SPARSE_INPUT,
-    R_TRANSFORM_SPARSE_OUTPUT,
+    # R_TRANSFORM_SPARSE_OUTPUT,
```

Comment:
Is this intentional?  If so why?

Reply:
To clearly indicate that the usage of the given constant was commented later in this file. When I run `black` - without the comment the constant is being removed, although the other comments remain in the file.